apps/web/src/design-system/template-button/TemplateButton.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { UnstyledButton, Group, Popover } from '@mantine/core';
+import styled from 'styled-components';
```

Comment:
Maybe its better to use emotion? To be consistent with mantine?

Reply:
Great catch! probably vscode autoimprot :(

Reply:
Thanks